ha_energy_reader.py

```
@@ -60,6 +60,31 @@ class EnergyDashboardConfig:
     battery_power: Optional[str] = None  # Combined: positive=charge, negative=discharge
     ev_power: Optional[str] = None
 
+    # #551 — battery ``power_config`` alternatives (HA ≥2026.x battery dialog).
+    # Two-sensor mode has NO single combined entity: HA stores
+    # ``power_config: {stat_rate_from, stat_rate_to}`` (from=discharge,
+    # to=charge) and battery_power stays None — the reader computes
+    # net = to − from. ``stat_rate_inverted`` flags a combined sensor whose
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)